### PR TITLE
dell/xps/13-9380: also import intel profile

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

**Comment by Mic92** [comment]
There is no AMD as an option: https://www.dell.com/en-uk/shop/laptops-notebooks-and-2-in-1-laptops/xps-13-laptop/spd/xps-13-9380-laptop

File: dell/xps/13-9380/default.nix [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 {
   imports = [
+    ../../../common/cpu/intel
```
Comment (Mic92) [inline dell/xps/13-9380/default.nix]: Since all of them also have NVME, I wonder if we should have a NVME profile?

Comment (kalbasit) [reply dell/xps/13-9380/default.nix]: You joined the 9380 fan-base? :grinning: 

Yea I think that's fair to say.

Comment (Mic92) [reply dell/xps/13-9380/default.nix]: I was not aware that this is a cult already. The screen is nice. The battery in my old laptop lasted a longer.

Comment (Mic92) [reply dell/xps/13-9380/default.nix]: The NVME profile might be worth another PR.

**Pull request merged**